<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FieldWidget</class>
 <widget class="QWidget" name="FieldWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>72</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>326</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="fieldLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLineEdit" name="fieldNameEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Field Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="typeBox">
      <item>
       <property name="text">
        <string>Integer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Float</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Text</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Datetime</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="primaryKeyBox">
      <property name="text">
       <string>Primary Key</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
